ts/src/header-validator/validate-json.ts

```diff
@@ -1208,8 +1212,34 @@ function aggregatableKeyValue(
     )
 }
 
-function aggregatableValues(ctx: Context, j: Json): Maybe<Map<string, number>> {
-  return keyValues(ctx, j, aggregatableKeyValue)
+function aggregatableValues(ctx: Context, j: Json): Maybe<AggregatableValue[]> {
+  return typeSwitch(ctx, j, {
+    object: (ctx, j) =>
+      struct(
+        ctx,
+        j,
+        {
+          aggregatableValue: (ctx, j) =>
+            keyValues(ctx, j, aggregatableKeyValue),
+          ...filterFields,
+        },
+        /*warnUnknown=*/ false
+      ).map((v) => [v]),
+    list: (ctx, j) =>
+      array(ctx, j, (ctx, j) =>
+        struct(
+          ctx,
+          j,
+          {
+            aggregatableValue: field('values', (ctx, j) =>
+              keyValues(ctx, j, aggregatableKeyValue)
+            ),
+            ...filterFields,
+          },
+          /*warnUnknown=*/ false
```

Comment:
We actually do want to warn about unknown fields here, as this is a struct with 3 known fields (`values, `filters`, `not_filters`).